shared/src/common/types/slc-function/slc-function-run-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type SLCFunctionRunRequestDto = {
+  payload?: string;
```

Comment:
1. Dto-s can have not required props?
2. It`s possible better rename "payload"?

Reply:
1. payload can be optional
2. payload is a good name 👍